File: C/Python/right_half_pyramid.py
#Printing Star Pattern right_half
num_rows = int(input("Enter the number of rows :"))

#Function to print right_half_pyramid
def right_half_pyramid(rows):
    #Loop to print rows
    for i in range(1, rows + 1):
        print("* "*i)

#Call the function
right_half_pyramid(num_rows)

File: C/right_half_pyramid.c
#include<stdio.h>
void main()
{
     int i,j;
    //For Loop
    for(i = 1; i<= 5; i++)
    {
        //Inner For Loop
        for(j = 1; j<=i; j++)
        {
            printf("* ");
        }
        printf("\n");
    }
}